baselines/disable-packing-test/test/gapic_messaging_v1beta1.ts.baseline

```diff
@@ -26,7 +26,7 @@ import * as messagingModule from '../src';
 import {PassThrough} from 'stream';
 
 import {protobuf, LROperation} from 'google-gax';
-
+import * as operationProtos from 'google-gax/build/protos/operations';
```

Comment:
Same feedback as above, let's not try to import things from nested paths

Reply:
cool! this is already exported from `gax`, so it can be imported from `google-gax` directly. Thank you!

Reply:
Waiting for the latest version of gax published.